bazel/cc_toolchains/clang_cc_toolchain_config.bzl

```diff
@@ -291,6 +291,32 @@ def _impl(ctx):
         ],
     )
 
+    x86_64_cpu_flags = feature(
+        name = "x86_64_cpu_flags",
+        enabled = True,
+        flag_sets = [
+            flag_set(
+                actions = all_compile_actions,
+                flag_groups = [flag_group(flags = [
+                    "-march=x86-64-v2",
+                ])],
+            ),
+        ],
+    )
+
+    aarch64_cpu_flags = feature(
```

Comment:
Are you wanting review right now? I might note, it's starting to feel odd to me how the variable declaration and use are so far apart -- I wonder though if this is more a comment about the factoring of this file in general, that it's not trying to keep variables and their use close together, use helper functions, etc.

Reply:
I was still testing things (and for some reason I couldn't find the button to mark it as a draft), but it seems to be working now.

On this specific comment, I'm just following the existing structure of the file. As you say, we could refactor this if desirable at some point. I don't have a strong opinion either way, but would prefer that to be a separate PR.